lib/rubocop/cop/rspec/expect_actual.rb

```diff
@@ -50,6 +50,7 @@ class ExpectActual < Base
           regexp
         ].freeze
 
+        SKIPPED_MATCHERS = %i[route_to be_routable].freeze
```

Comment:
Since we are planning to extract the rspec-rails specific parts from rubocop-rspec, I don’t think we should hardcode these matchers names. Can we either have (the non-existing) rubocop-rspec-rails inject them, or maybe just add a `SkippedMatchers` to the configuration?

Reply:
Could we wait to see how the rails extraction takes shape before doing that? It's true that this is rails-specific, but it's also highly unlikely to matter, since we'd only miss adding offenses for non-rails matchers that happened to be named "route_to" or "be_routable".